x-pack/test_serverless/api_integration/test_suites/observability/ai_assistant/tests/knowledge_base/helpers.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MachineLearningProvider } from '@kbn/test-suites-xpack/api_integration/services/ml';
+import { SUPPORTED_TRAINED_MODELS } from '@kbn/test-suites-xpack/functional/services/ml/api';
+
+export const TINY_ELSER = {
+  ...SUPPORTED_TRAINED_MODELS.TINY_ELSER,
+  id: SUPPORTED_TRAINED_MODELS.TINY_ELSER.name,
+};
+
+export async function deleteKnowledgeBaseModel(ml: ReturnType<typeof MachineLearningProvider>) {
```

Comment:
Moved over because we can't delete system indices in MKI so it was removed from this function.

Reply:
we cannot create a user with the `kibana_system` role?

Reply:
Unfortunately, it's not possible right now.  We cannot create that role.  I will send you more info.